core/php/Utilities/StringUtility.php

```diff
@@ -205,6 +205,35 @@ function az09($string, $preserve = "", $strToLower = TRUE) {
     $charGroup[] = array("2","²");
     $charGroup[] = array("3","³");
     $charGroup[] = array("and","&");
+    $charGroup[] = array("a","А","а");
+    $charGroup[] = array("b","Б","б");
+    $charGroup[] = array("v","В","в");
+    $charGroup[] = array("g","Г","г");
+    $charGroup[] = array("d","Д","д");
+    $charGroup[] = array("e","Е","е","Ё","ё");
+    $charGroup[] = array("zh","Ж","ж");
+    $charGroup[] = array("z","З","з");
+    $charGroup[] = array("i","И","и","Ы","ы");
+    $charGroup[] = array("y","Й","й","Ъ","ъ");
+    $charGroup[] = array("k","К","к");
+    $charGroup[] = array("l","Л","л");
+    $charGroup[] = array("m","М","м");
+    $charGroup[] = array("n","Н","н");
+    $charGroup[] = array("o","О","о");
+    $charGroup[] = array("p","П","п");
+    $charGroup[] = array("r","Р","р");
+    $charGroup[] = array("s","С","с");
+    $charGroup[] = array("t","Т","т");
+    $charGroup[] = array("u","У","у");
```

Comment:
In theory it should be possible to keep uppercase characters uppercase controlled by the 3rd method argument `$strToLower`. but i have to admit that sliMpd's usage always converts the string to lowercase.

Anyway - to keep the method as good as possible it would be great if you can split replacements to a lowercase and an uppercase version.
So instead of adding the line `$charGroup[] = array("u","У","у");` please add those chars to the existing lines
```php
$charGroup[] = array("u","ù","ú","û","ü","у");
$charGroup[] = array("U","Ù","Ú","Û","Ü","У");
```

Reply:
Done